catalog-info.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: oauth2-proxy
+  annotations:
+    github.com/project-slug: philips-forks/oauth2-proxy
+    philips.com/techdocs-builder: local
+  description: TBD
```

Comment:
Maybe add some info from the readme file:
A reverse proxy and static file server that provides authentication using Providers (Google, GitHub, and others) to validate accounts by email, domain or group.

Note: This repository was forked from [bitly/OAuth2_Proxy](https://github.com/bitly/oauth2_proxy) on 27/11/2018. Versions v3.0.0 and up are from this fork and will have diverged from any changes in the original fork. A list of changes can be seen in the [CHANGELOG](https://github.com/philips-forks/oauth2-proxy/blob/master/CHANGELOG.md).